drivers/tests/UNITTESTS/PwmOut/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+cmake_minimum_required(VERSION 3.0.2)
```

Comment:
shouldn't this be in our main unittests cmake? 

I noticed this one is 3.0.2 other is 3.19?